### PR TITLE
[stable/filebeat] Support filebeat 7.0.1

### PR DESCRIPTION
**Pull request by Mokto**
Signed-off-by: Theo Mathieu <tmathieu.github@fastmail.com>

#### What this PR does / why we need it:
Support elasticsearch 7.0.0

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/filebeat/values.yaml [hunk 1]
```diff
@@ -1,24 +1,13 @@
 image:
   repository: docker.elastic.co/beats/filebeat-oss
-  tag: 6.7.0
+  tag: 7.0.1
   pullPolicy: IfNotPresent
 
 config:
-  filebeat.config:
```
Comment (sstarcher) [inline stable/filebeat/values.yaml]: has this been removed in filebeat 7.0.1?

File: stable/filebeat/Chart.yaml [hunk 2]
```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 description: A Helm chart to collect Kubernetes logs with filebeat
 icon: https://www.elastic.co/assets/blt47799dcdcf08438d/logo-elastic-beats-lt.svg
 name: filebeat
-version: 1.7.0
```
Comment (sstarcher) [inline stable/filebeat/Chart.yaml:5]: What's the procedure here should we bump this to 2.0.0 since the major version of filebeat is increasing?

Comment (Mokto) [reply stable/filebeat/Chart.yaml:5]: Yeah, I was not sure about this one.

It's kind of a breaking change, but it's your call

File: stable/filebeat/values.yaml (hunk 1 above)
Comment (Mokto) [reply stable/filebeat/values.yaml]: Yes but let me confirm that tomorrow. 

I'll do some more checks.

Comment (Mokto) [reply stable/filebeat/values.yaml]: You were right, only filebeat.config.prospectors has been removed. 

Thanks !

File: stable/filebeat/Chart.yaml (hunk 2 above)
Comment (sstarcher) [reply stable/filebeat/Chart.yaml:5]: I would think we want to bump the major version.

File: stable/filebeat/values.yaml [hunk 3]
```diff
@@ -1,15 +1,10 @@
 image:
   repository: docker.elastic.co/beats/filebeat-oss
-  tag: 6.7.0
+  tag: 7.0.1
   pullPolicy: IfNotPresent
 
 config:
   filebeat.config:
-    prospectors:
```
Comment (sstarcher) [inline stable/filebeat/values.yaml:8]: so now that this is removed do inputs get loaded automatically and we don't need this info?

**Comment by sc7565** [comment]
Thanks for update, can we get this merged

File: stable/filebeat/Chart.yaml (hunk 2 above)
Comment (Mokto) [reply stable/filebeat/Chart.yaml:5]: Done

File: stable/filebeat/values.yaml (hunk 3 above)
Comment (Mokto) [reply stable/filebeat/values.yaml:8]: Yes, I have also tested it.

**Pull request merged**